### ISSUE TITLE
Support a "solid" archive format

### ISSUE THREAD

**Issue by lapo-luchini** [mtkennerly/ludusavi#317]
### What's your idea?

Some games have got very similar save files, e.g. Terraria keeps a single previous version as `.bak` (which maybe we could just skip since it is redundant with Ludusavi options for keeping multiple backups, but that's another issue), but using ZIP file format have the files occupy the whole space each time:
![image](https://github.com/mtkennerly/ludusavi/assets/420454/5b4758ba-7a2c-4e50-9c06-19cbd89b36ab)
Using any "solid" compression format, such as `.7z` or even `.tar.xz` this would reduce the used space by almost 50%.

**Comment by sluedecke** [comment]
I wonder, isn't your backup zipped alrady - the filename suggests so?

Mine are:

![image](https://github.com/mtkennerly/ludusavi/assets/7710530/19eb611a-e9f8-42d1-9e88-6fe9548f05ef)

**Comment by lapo-luchini** [comment]
Well yes, that's why 7-zip GUI is showing file size 7 MiB and packed size of 2.2 MiB. :innocent: 
But if the file was solid, the total size would be 2.2 MiB, not 4.4 MiB (as the two files are very similar and thus the second one would re-use most of the compressed chunks in the first).

**Comment by GlobalCBWK** [comment]
True, supporting .7z format could significantly reduce storage requirements.
